packages/fints/src/types.ts

```diff
@@ -150,6 +152,24 @@ export interface SEPAAccount {
      * The account's BLZ used together with the account number before SEPA.
      */
     blz: string;
+
+    accountOwnerName?: string;
```

Comment:
Would be awesome if you could add some documentation here.

Reply:
I found https://www.hbci-zka.de/dokumente/spezifikation_deutsch/fintsv3/FinTS_3.0_Messages_Geschaeftsvorfaelle_2015-08-07_final_version.pdf but I don't know how to read it. If you know maybe we can have a zoom call. Afterwards I would be happy to add it here.